lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -54,6 +54,18 @@ class GoogleCloudOutput < BufferedOutput
     config_param :zone, :string, :default => nil
     config_param :vm_id, :string, :default => nil
 
+    # List of parsed fields whose values are sent as labels instead of
+    # becoming part of the struct payload.
+    #
+    # This is specified as a JSON array where the 'name' is the field name
+    # parsed by fluentd and the 'value' is the name of the label to send.
```

Comment:
Maybe change 'name' to 'map key' and 'value' to 'map value' to make it more clear?
On my first reading I thought we are just applying static values as label values, that's why I made that first comment below.